internal/controllers/helpers/handlers.go

```diff
@@ -149,3 +151,60 @@ func ErrorHandler(c *fiber.Ctx, err error, logger zerolog.Logger, isProduction b
 		"message": err.Error(),
 	})
 }
+
+type DeviceAttributeType string
+
+const (
+	MPG                 DeviceAttributeType = "mpg"
```

Comment:
```suggestion
	Mpg                 DeviceAttributeType = "mpg"
```
just for consistency, or change the other one to be `MPGHighway`